shared-doc/attributes.adoc

```diff
@@ -37,6 +39,9 @@ ifdef::EAPXPRelease[]
 :productNameFull: Red Hat JBoss Enterprise Application Platform expansion pack
 :productVersion: 3.0
 :DocInfoProductNumber: 7.4
+:githubRepoUrl: https://github.com/jboss-developer/jboss-eap-quickstarts
```

Comment:
this is defined already a few lines below in the IF product release or xp release, and thus overrides this definition

Reply:
fixed